src/components/BottomNav.js

```diff
@@ -10,9 +10,7 @@ import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
 
 const useStyles = makeStyles(theme => ({
   root: {
-    minWidth: 375,
-    maxWidth: 3000,
-    height: 30,
+    minWidth: 375, // the width of an ipone x
   },
 }));
```

Comment:
I think we can get rid of this, just because I messed around with resizing the `BottomNav` and it's been keeping it's shape without these styles.